qiskit/opflow/primitive_ops/tapered_pauli_sum_op.py

```diff
@@ -34,6 +34,8 @@
 class TaperedPauliSumOp(PauliSumOp):
     """Class for PauliSumOp after tapering"""
 
+    primitive: SparsePauliOp
+
```

Comment:
Is this class level var intentional or left over from testing?

Reply:
Thanks for your review. From [the test result](https://dev.azure.com/qiskit-ci/qiskit-terra/_build/results?buildId=27193&view=logs&jobId=7684b0c9-af38-5400-184f-6880bd8a4dd7&j=7684b0c9-af38-5400-184f-6880bd8a4dd7&t=9d95188b-4e59-58d8-f7c4-3f1fbd4209ad), I added this line.
I am not sure why this line is needed, but it may be false positive of pylint.

Reply:
I note that PyCharm says this about the TaperedPauliSumOp init method `Signature is not compatible to __new__` which I imagine might not help things for the pylint.

I will note that on that if you use super() instead of self on the line that lint complains about then it passes ie 
```
pauli_sum = PauliSumOp(super().primitive, self.coeff)
```
Quite why primitive is an issue but coeff, that looks like similar code in that regard is not, I did not look further. I imagine that getting the init and new issue corrected might fix things for self.  Having to have that dummy class member of the same name does not seem right. Better to just suppress the lint failure if the above does not seem ok to you - though arguably the user of super() there seems odd too so one might want to comment if you chose to use that.

Reply:
@ikkoham In order to have this fix get into the imminent release I went ahead and changed the code to suppress the pylint error rather than have the dummy class instance var. Things can always be improved later if you find a better way such that pylint is happy - but for now it fixes the symmetry bug and we need to get this fix out.

Reply:
I also like your solution. Thanks a lot.